src/gha_runner/__main__.py

```diff
@@ -142,6 +146,9 @@ def main():  # pragma: no cover
     provider = os.environ.get("INPUT_PROVIDER")
     if provider is None:
         raise Exception("Missing required input variable INPUT_PROVIDER")
+    # Set the default timeout to 20 minutes
+    gh_timeout = int(os.environ.get("INPUT_GH_TIMEOUT", 1200))
+    gh_timeout = 1200
```

Comment:
```suggestion
    gh_timeout = int(os.environ.get("INPUT_GH_TIMEOUT", 1200))
```

I'll just self-merge this suggestion; pretty sure it was needed!